### ISSUE TITLE
[1.18] NPE from CMLogger on game launch

### ISSUE THREAD

**Issue by Charnuz** [Frinn38/Custom-Machinery#48]
```
java.lang.NullPointerException: Cannot invoke "org.apache.logging.log4j.core.Appender.getName()" because "appender" is null
	at org.apache.logging.log4j.core.config.AppenderControl.<init>(AppenderControl.java:56) ~[log4j-core-2.17.0.jar%2342!/:2.17.0] {}
	at org.apache.logging.log4j.core.config.LoggerConfig.addAppender(LoggerConfig.java:206) ~[log4j-core-2.17.0.jar%2342!/:2.17.0] {}
	at fr.frinn.custommachinery.common.util.CMLogger.init(CMLogger.java:81) ~[Custom-Machinery-1.18.2-0.6.0.jar%2372!/:1.18.2-0.6.0] {re:classloading}
	at fr.frinn.custommachinery.CustomMachinery.<init>(CustomMachinery.java:56) ~[Custom-Machinery-1.18.2-0.6.0.jar%2372!/:1.18.2-0.6.0] {re:classloading}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:67) ~[javafmllanguage-1.18.2-40.1.22.jar%23125!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[fmlcore-1.18.2-40.1.22.jar%23124!/:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}
```

[Full crash log](https://pastebin.com/DuezWCD5)

**Comment by redgoblin88** [comment]
i get the same crash with only

forge: 40.1.25
CM: 1.18.2-0.6.0

**Comment by Frinn38** [comment]
I see the issue, that happen if the forge debug.log file is  disabled, which is the default behaviour of the cursefprge app and probably some other launchers.
Thanks for the report, that will be fixed soon.

**Comment by Frinn38** [comment]
Fixed in latest version (1.18.2-0.6.1)
